spec/tasks/broadcasts_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe "Broadcasts tasks", type: :task do
+  let(:service) { Broadcasts::WelcomeNotification::Generator }
+
+  let_it_be_readonly(:default_config_date) { SiteConfig.welcome_notifications_live_date }
+
+  before do
+    SiteConfig.welcome_notifications_live_date = 1.week.since
```

Comment:
might be helpful to add comments next to the dates here so that this spec is easy to parse (maybe it's just me but i always have to wrangle my head around dates 🙈)

Reply:
I went ahead and extract it to a more semantic variable.